client/asset/btc/btc.go

```diff
@@ -775,12 +797,7 @@ func openSPVWallet(cfg *BTCCloneCFG) (*ExchangeWalletSPV, error) {
 		return nil, err
 	}
 
-	var peers []string
```

Comment:
I removed the code related to peer configuration since there is no `ConfigOpt` for the peers.